.github/scripts/run.sh

```diff
@@ -36,13 +36,15 @@ sudo nvidia-smi -ac ${GPU_FREQUENCY}
 export CUDA_VISIBLE_DEVICES="${GPU_LIST}"
 export GOMP_CPU_AFFINITY="${CORE_LIST}"
 
-echo "Running benchmark with filter: \"${BENCHMARK_FILTER}\""
+# echo "Running benchmark with filter: \"${BENCHMARK_FILTER}\""
+echo "Running check_lazy.py"
 
 # Run the benchmark
-for c in $(seq 1 $NUM_ITER); do
-    taskset -c "${CORE_LIST}" pytest test_bench.py -k "${BENCHMARK_FILTER}" \
-            --benchmark-min-rounds "${NUM_ROUNDS}" \
-            --benchmark-json ${DATA_DIR}/${DATA_JSON_PREFIX}_${c}.json
-done
+# for c in $(seq 1 $NUM_ITER); do
+#     taskset -c "${CORE_LIST}" pytest test_bench.py -k "${BENCHMARK_FILTER}" \
+#             --benchmark-min-rounds "${NUM_ROUNDS}" \
+#             --benchmark-json ${DATA_DIR}/${DATA_JSON_PREFIX}_${c}.json
+# done
+taskset -c "${CORE_LIST}" python check_lazy.py --output_file ${DATA_DIR}/sweep.out
```

Comment:
You will need to add to `https://github.com/pytorch/benchmark/blob/main/.github/workflows/lazy-tensor-nightly.yml` on main branch to upload the result as github actions artifact.

Reply:
Sure, thanks for the suggestion. We will rely on the print first, and then move on to this route.